Sources/xcproj/PBXBuildRule.swift

```diff
@@ -0,0 +1,134 @@
+import Foundation
+
+/// A PBXBuildRule is used to specify a method for transforming an input file in to an output file(s).
+final public class PBXBuildRule: PBXObject, Equatable {
+
+    // MARK: - Attributes
+
+    /// Element compiler spec.
+    public var compilerSpec: String
+
+    /// Element file patterns.
+    public var filePatterns: String?
+
+    /// Element file type.
+    public var fileType: String
+
+    /// Element is editable.
+    public var isEditable: UInt
```

Comment:
Can this take any other values than 1 or 0? Otherwise we can make it a Bool

Reply:
I believe it's only ever a 1 or 0. There were some existing seemingly Bool properties were using (U)Int but definitely happy to change it.

Reply:
Yeah, those other ones could be changed in 2.0 as it's a breaking change

Reply:
https://github.com/xcodeswift/xcproj/issues/204